src/FrameReflower/Text.php

```diff
@@ -386,13 +389,35 @@ function reflow(BlockFrameDecorator $block = null)
 
         // Determine the text height
         $style = $frame->get_style();
+        $font_metrics = $this->getFontMetrics();
+
+        // Handle text transform and white space
+        $text = $this->pre_process_text($frame->get_text());
+
+        // map text to fonts based on supported Unicode range
+        $families = array_map(
+            function ($value) {
+                return trim($value, " '\"");
+            },
+            explode(",", $style->get_specified("font_family"))
+        );
+        $charMapping = $font_metrics->mapFontsToText($text, $families, $font_metrics->getType($style->font_weight . ' ' . $style->font_style), 1);
+        if (isset($charMapping[0])) {
+            if ($charMapping[0]["length"] !== 0) {
+                $frame->split_text($charMapping[0]["length"]);
+                $text = $this->pre_process_text($frame->get_text());
```

Comment:
`$this->pre_process_text()` should no be needed here, already handled above.

Reply:
The frame text hasn't yet been set to the processed version. I think we need to move the `set_text`, which is currently below this logic, before this block so that the split occurs in the correct location after processing. Even so, the split frames will have their text run through pre-processing again, which is not ideal.

Reply:
I think you are right, yes.